src/System.Private.CoreLib/src/System/Reflection/Emit/TypeBuilder.cs

```diff
@@ -807,7 +788,7 @@ public override RuntimeTypeHandle TypeHandle
             get { throw new NotSupportedException(SR.NotSupported_DynamicModule); }
         }
 
-        public override string FullName
+        public override string? FullName
```

Comment:
Similar question here.  It'd be good to validate if you haven't already whether null can actually be returned from the override.  In places where we won't return null, it'd be good to narrow the constraint.

Reply:
In this case calling method `TypeNameBuilder.ToString(this, TypeNameBuilder.Format.FullName)` can return null
```chsarp
internal static string? ToString(Type type, Format format)
        {
            if (format == Format.FullName || format == Format.AssemblyQualifiedName)
            {
                if (!type.IsGenericTypeDefinition && type.ContainsGenericParameters)
                    return null;
            }
```
But as i mentioned you once there were case where properties obviously not returning null but because of overriding from parent compiler not allowing remove '?'. I have reported that issue here: https://github.com/dotnet/roslyn/issues/23268#issuecomment-482306719

Reply:
> In this case calling method TypeNameBuilder.ToString(this, TypeNameBuilder.Format.FullName) can return null

Ok, thanks.

> because of overriding from parent compiler not allowing remove '?'

Thanks.  For cases where you should believe you should be able to constraint it but hit a warning, you can suppress the warning, e.g. as @safern did in https://github.com/dotnet/corefx/pull/37014/commits/29bb48021685c2f990f7f60eb35e6304d0bc3bd0.

Reply:
Great thanks